devicetypes/smartthings/zigbee-valve.src/zigbee-valve.groovy

```diff
@@ -88,38 +90,57 @@ def parse(String description) {
         } else if (event.name == "powerSource") {
             event.value = event.value.toLowerCase()
         }
-
         sendEvent(event)
     }
     else {
         def descMap = zigbee.parseDescriptionAsMap(description)
         if (descMap.clusterInt == CLUSTER_BASIC && descMap.attrInt == BASIC_ATTR_POWER_SOURCE){
             def value = descMap.value
             if (value == "01" || value == "02") {
-                sendEvent(name: "powerSource", value: "mains")
+                sendEvent(name: "powerSource", value: "Mains")
```

Comment:
All of these values should be left lowercase. The capability and ocf mapping require lowercase.